client/header/activity-panel/panels/inbox/inbox-panel.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './inbox.scss';
+import NotesPanel from '../../../../inbox-panel';
+import { AbbreviatedCard } from './abbreviated-card';
+import { cards } from './cards';
+
+export const InboxPanel = ( { notifications } ) => {
+	const getCardByName = ( name ) => {
+		return cards.find( ( card ) => card.name === name );
+	};
+
+	return (
+		<div className="woocommerce-notification-panels">
+			{ notifications.length > 0 && (
+				<div className="woocommerce-abbreviated-cards">
+					{ notifications.map( ( { count, critical, name } ) => {
+						const card = getCardByName( name );
+						if ( ! card ) {
+							return null;
+						}
+						const { content, href, icon, title } = card;
+						return (
+							<AbbreviatedCard
+								content={ sprintf( content, count ) }
+								critical={ critical ?? 0 }
```

Comment:
Does this need to be a prop? Can it just be part of the content? This doesn't feel like a generic property to me since (right now) AFAICT it's only for the "things to do next" item and the label isn't customizable (it always says "X critical alerts").

Reply:
I really like the idea! That'd make the component more generic and simple 👍 
This change was added in the commit 373e19e